packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -187,6 +187,16 @@ export type AzureCommunicationChatAdapterArgs = {
     threadId: string;
 };
 
+// @public
+export interface BackgroundImage {
+    extension?: string;
+    name?: string;
```

Comment:
I assume this will be used for tooltips? Do we need name for anything else?

Reply:
yeah, these are only for tooltips to show the name on hover.

Reply:
This could be interpreted as filename (which is likely not what the desired tooltip name would be).
Rename to `tooltipName`/`displayName`/`friendlyName`/`nameHint`?